tools/content-validation/docs/deployment.md

```diff
@@ -0,0 +1 @@
+// *TODO*
```

Comment:
Should this include instructions for running the tool periodically in a pipeline?